modules/virt-hot-plugging-a-virtual-disk-using-the-web-console.adoc

```diff
@@ -6,19 +6,21 @@
 [id="virt-hot-plugging-a-virtual-disk-using-the-web-console{context}"]
 = Hot-plugging a virtual disk using the web console
 
-Hot-plug virtual disks that you want to attach to a virtual machine instance (VMI) while a virtual machine is running.
+Hot-plug virtual disks that you want to attach to a virtual machine instance (VMI) while a virtual machine is running. When you hot-plug a virtual disk, it remains attached to the VMI until you hot-unplug it.
 
 .Prerequisites
 * You must have a running virtual machine to hot-plug a virtual disk.
 
 .Procedure
 
-. Click *Virtualization* -> *VirtualMachines* from the side menu.
+. Click *Virtualization* -> *Virtual Machines* from the side menu.
```

Comment:
This should be **VirtualMachines** to match the UI

Reply:
Done.